src/DGtal/geometry/surfaces/estimation/PlaneProbingTetrahedronEstimator.h

```diff
@@ -0,0 +1,332 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+#pragma once
+
+/**
+ * @file
+ * @author Jocelyn Meyron (\c jocelyn.meyron@liris.cnrs.fr )
+ * Laboratoire d'InfoRmatique en Image et Systemes d'information - LIRIS (CNRS, UMR 5205), CNRS, France
+ *
+ * @date 2020/12/04
+ *
+ * Header file for module PlaneProbingTetrahedronEstimator.cpp
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(PlaneProbingTetrahedronEstimator_RECURSES)
+#error Recursive header files inclusion detected in PlaneProbingTetrahedronEstimator.h
+#else // defined(PlaneProbingTetrahedronEstimator_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define PlaneProbingTetrahedronEstimator_RECURSES
+
+#if !defined PlaneProbingTetrahedronEstimator_h
+/** Prevents repeated inclusion of headers. */
+#define PlaneProbingTetrahedronEstimator_h
+
+//////////////////////////////////////////////////////////////////////////////
+// Inclusions
+#include <iostream>
+#include <vector>
+#include "DGtal/base/Common.h"
+#include "DGtal/geometry/helpers/PlaneProbingEstimatorHelper.h"
+#include "DGtal/geometry/surfaces/estimation/PlaneProbingNeighborhood.h"
+#include "DGtal/kernel/CPointPredicate.h"
+//////////////////////////////////////////////////////////////////////////////
+
+namespace DGtal
+{
+    /**
+     * Probing mode for plane-probing estimators.
+     */
+    enum class ProbingMode
+    {
+        H,
```

Comment:
Perhaps add short description for each ProbingMode ?

Reply:
I added a description for the modes as well as for the ProbingMode enum.